packages/javascript/bh-shared-ui/src/test-utils.jsx

```diff
@@ -18,10 +18,13 @@
 import React from 'react';
```

Comment:
these changes are to match bhe `test-utils.jsx` as much as possible. It main intention is adding support for testing customHooks with providers